include/main.h

```diff
@@ -37,7 +37,7 @@ struct Main
 
     /*0x439*/ u8 oamLoadDisabled:1;
     /*0x439*/ u8 inBattle:1;
-    /*0x439*/ u8 field_439_x4:1;
+    /*0x439*/ u8 isFrontierBattle:1;
```

Comment:
This name seems wrong. This is set for any wireless link multi battle as long as someone has received the Frontier Pass before. `FLAG_SYS_FRONTIER_PASS` isn't cleared upon leaving Battle Frontier